service_capacity_modeling/hardware/profiles/shapes/aws.json

```diff
@@ -725,63 +725,71 @@
       "cpu_ghz": 3.1,
       "ram_gib": 15.71,
       "net_mbps": 500,
-      "drive": null
+      "drive": null,
+      "platforms": "Aurora MySQL,Aurora PostgreSQL"
```

Comment:
Can you just make this a list instead of a string, e.g. ["aurora-mysql", "aurora-postgresql"] ?

Reply:
Did in a followup.